### PR TITLE
reduce repetitive getDefaultNamespace() and controllerName() function call in Router

### PR DESCRIPTION
**Pull request by samsonasik**
**Checklist:**
- [x] Securely signed commits

### CODE REVIEW

File: system/Router/Router.php [hunk 1]
```diff
@@ -548,11 +548,13 @@ public function autoRoute(string $uri)
 			$this->params = $segments;
 		}
 
+		$defaultNameSpace = $this->collection->getDefaultNamespace();
```
Comment (michalsn) [inline system/Router/Router.php:551]: I know this is a bit picky, but can we make it `defaultNamespace`?

Comment (MGatner) [reply system/Router/Router.php:551]: I, also, think this is picky and would like to see it changed 😅

Comment (samsonasik) [reply system/Router/Router.php]: I've updated to `defaultNamespace` 👍

**Comment by michalsn** [comment]
Thank you!

**Pull request merged**